distribution/scripts/ballerina/xml_csv_transformation/main.bal

```diff
@@ -0,0 +1,50 @@
+import ballerina/data.xmldata;
+import ballerina/http;
+
+configurable string epKeyPath = ?;
+configurable string epKeyPassword = ?;
+
+type Order record {|
+    string symbol;
+    string buyerID;
+    float price;
+    int volume;
+|};
+
+type Invoice record {|
+    Order[] 'order;
```

Comment:
```suggestion
    Order[] orders;
```

Reply:
It is not possible to do this because of the payload that I get have the field `order` to represent a set of orders and I'm not in charge of changing the payload